kernel.go

```diff
@@ -373,6 +369,62 @@ func handleExecuteRequest(ir *classic.Interp, receipt msgReceipt) error {
 	return receipt.Reply("execute_reply", content)
 }
 
+// doEval evaluates the code in the interpreter. This function captures an uncaught panic
+// as well as the value of the last statement/expression.
+func doEval(ir *classic.Interp, code string) (val interface{}, err error) {
```

Comment:
We should use returns explicitly instead of named returns.  Makes it more readable.

Reply:
In `doEval` there is a special case because we need to capture the result of `recover()` in a return parameter so I think we are stuck with a named parameter in this case but all of the other returns in this function are changed to be explicit.

Reply:
Gotcha.  Well, I would just make sure all the other returns are explicit, even if you have them named in the function definition.